controlplane/src/core/bufservices/subgraph/checkSubgraphSchema.ts

```diff
@@ -295,7 +301,8 @@ export function checkSubgraphSchema(
       featureId: 'breaking-change-retention',
     });
 
-    const limit = changeRetention?.limit ?? 7;
+    let limit = changeRetention?.limit ?? 7;
+    limit = clamp(namespace?.checksTimeframeInDays ?? limit, 1, limit);
```

Comment:
If it's configured, use the configured value or else use the org limit

Reply:
Could there not be cases where the organization limit is changed?

Reply:
But that shouldn't cause a problem right?

Reply:
It shouldn't, the only thing would be that if they downgrade, this configuration would override the new plan